src/controllers/import.js

```diff
@@ -137,8 +196,15 @@ function ImportController(context) {
     const { 'x-api-key': importApiKey, 'user-agent': userAgent } = headers;
 
     try {
-      // The API scope imports.write is required to create a new import job
-      validateAccessScopes([SCOPE.WRITE]);
+      try {
+        // The API scope imports.write is required to create a new import job
+        validateAccessScopes([SCOPE.WRITE]);
+      } catch (error) {
+        log.error(`Missing imports.write scope for creating a new import job: ${error.message}`);
+        // Check if the user has the write_all_domains API scope.
+        // This scope is restricted to a few users.
+        validateAccessScopes([SCOPE.WRITE_ALL_DOMAINS]);
```

Comment:
The method `validateAccessScope` throws an error in case the scopes do not match. So we might need to use this check instead:
```
if (!scopes.some((scope) => scope.name === SCOPE.WRITE_ALL_DOMAINS)) {
   validateUrlsByDomain(urls);
}
```